scripts/parachain-launch/README.md

```diff
@@ -0,0 +1,24 @@
+# parachain-launch
+
+
+
+# Introduction
+
+[Parachain-launch](https://github.com/open-web3-stack/parachain-launch) is a script that generates a docker compose file allowing you to launch a testnet of multiple blockchain nodes.
+
+The output directory already contains a generated docker-compose file so you can spin a testnet up without needing to install `parachain-launch`
+
+To get the testnet up and running run the below command
+
+```
+docker-compose -f output/docker-compose.yml up -d
+```
+
+To regenerate the compose file, first ensure `parachain-launch` is installed and run the following command
+
+```
+parachain-launch generate statemine.yml
+```
```

Comment:
I think this should be reverted. We also should not commit the docker compose file. You should describe the way on how to convert the statemine.yml file to the docker compose file and then how to run this. Then we can remove the entire `output` folder.

Reply:
I would have thought having the docker-compose file added might have been simpler since its one less thing to do, but I understand it might be simpler to only have the config file in the repo